bash/cmd-line-searches/howmuch

```diff
@@ -1,10 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-#================================================================================================================
+#=======================================================================================================
 #title           :howmuch
-#description     :Presents a list of size of files in current directory human readably, and sorted descending
+#description     :Lists sizes of all files in specified directory human readably, and sorted descending
 #author          :dharvey-consbio
 #usage           :see https://github.com/consbio/cookbook/tree/master/bash/cmd-line-searches/#howmuch
-#=================================================================================================================
+#=======================================================================================================
 
-du -chs .[!.]* * |sort -r --human-numeric-sort |head -11
+export GLOBIGNORE=".:.."
+
+if [[ -z $1 ]] ; then
+  export searchin=.
+else
+  export searchin="$1"
+  echo "Searching in $searchin"
```

Comment:
I don't think "search" is really the accurate term here.

Reply:
How about "checking" instead of "searching in"?

Reply:
Or, "Listing files in"?

Reply:
Or "Summarizing <dir>"? Though, personally I'd just leave that off and just print the directly name, and call the variable simply `dir`.

Reply:
I think I like "Summarizing" better. It makes it clear that it is a message. Otherwise, to my eye, the directory tends to get confused with the output.

Reply:
Printing the dir might not be necessary at all. Consider all the other utilities (ls, du, etc.) that don't print the directory name (presumably because you just typed it, you already know.

Reply:
Good point. I guess I just like a little user-friendly verbosity. But I'll remove it.

Reply:
(I also renamed `searchin` to `checkdir` since `dir` is a linux command)